### ISSUE TITLE
Can't get it working

### ISSUE THREAD

**Issue by gubagu** [nkalinov/ng2-datetime#39]
Hi there, First great component ! Unfortunately I can't get it working. I thought something was wrong with my app settings because I use webpack. I got these errors:

```
ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(51,17): error TS7006: Parameter '_' implicitly has an 'any' type.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(115,21): error TS7006: Parameter 'e' implicitly has an 'any' type.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(140,37): error TS2304: Cannot find name '$'.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(142,36): error TS7006: Parameter 'e' implicitly has an 'any' type.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(157,19): error TS2304: Cannot find name '$'.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(161,27): error TS2304: Cannot find name 'jQuery'.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(162,37): error TS2304: Cannot find name '$'.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(164,47): error TS7006: Parameter 'e' implicitly has an 'any' type.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(189,19): error TS2304: Cannot find name '$'.

ERROR in user\app\node_modules\ng2-datetime\src\ng2-datetime\ng2-datetime.ts
(223,17): error TS7006: Parameter 'obj' implicitly has an 'any' type.
```

Then I cloned the repo and run npm && npm start but got this:
```(index):25 Error: (SystemJS) TypeError: _angular_core.createPlatformFactory is not a function(…)```

Any ideas what that could be ?

Thanks in advance,
Guby

**Comment by fmccready** [comment]
Some of those errors might be because you have noImplicitAny set to true in your tsconfig.json file.

**Comment by nkalinov** [comment]
Hi @gubagu ,
1. Errors with implicitly set parameters can be avoided with `noImplicitAny=false`. I will fix this in 1.1.5.
2. `Cannot find name...` errors are because you haven't installed the type definitions, though those are installed automatically when you installed the package, yours aren't  -> `typings install --global --save jquery`
